src/lookups/lookup.jade

```diff
@@ -0,0 +1,14 @@
+.slds-lookup
+  .slds-form-element
+    label.slds-form-element__label
+      ng-content(select="[label]")
```

Comment:
A suggestion here, would be to introduce an `nglLookupHeading` directive and do `select="[nglLookupHeading]"`. That way you can explicitly add the `slds-form-element__label` class to it as well as use the `for` attribute to relate it to the input field.